app/stac_api/pgtriggers.py

```diff
@@ -369,23 +394,18 @@ class UpdateCollectionUpdateIntervalTrigger(pgtrigger.Trigger):
         '''
 
     return [
-        pgtrigger.Trigger(
-            name="add_item_auto_variables_trigger",
-            operation=pgtrigger.Insert,
-            when=pgtrigger.Before,
-            func=AUTO_VARIABLES_FUNC
-        ),
-        pgtrigger.Trigger(
-            name="update_item_auto_variables_trigger",
-            operation=pgtrigger.Update,
-            when=pgtrigger.Before,
-            condition=pgtrigger.Condition('OLD.* IS DISTINCT FROM NEW.*'),
-            func=AUTO_VARIABLES_FUNC
-        ),
+        *auto_variables_triggers('item'),
+        *generate_child_triggers('collection', 'Item'),
         UpdateCollectionExtentTrigger(
             name='update_item_collection_extent_trigger',
             operation=pgtrigger.Update,
-            condition=pgtrigger.Condition('OLD.* IS DISTINCT FROM NEW.*')
+            # condition=pgtrigger.Condition('OLD.* IS DISTINCT FROM NEW.*')
+            condition=pgtrigger.Condition(
```

Comment:
yes good improvement here thank you, that should prevent us from useless extent aggregation in the future :+1: